app.js

```diff
@@ -16,19 +17,51 @@ const login = require('./routes/login')
 
 const app = express()
 
+let currentElectionInfo = {
+	id: -1,
+	checked: moment(),
+	isStale: true
```

Comment:
So we can manually force a refresh. I picture us doing this every time the `/elections` route is hit, which makes sure that changes to an election are applied instantly to requests, rather than having to wait for the old data to time out.